newsfragments/30083.significant.rst

```diff
@@ -0,0 +1,13 @@
+Non-fixed cron timetables no longer account for changing in timezone.
+
+Given a cron expression resulting in an interval-based schedule, such as
+``0 9-16 * * *``, job after a timezone change (e.g. entering or exiting daylight
+saving time) used to be scheduled with an offset to the change in timezone. For
+example, when operating in *America/New_York*, the above schedule would result
+in a run at 10am on Monday morning after entering DST, instead of the previous
+9am.
+
+This was intentional to account for the change in timezone introduced in Airflow
+2.2, but caused confusion to several use cases. The functionality is now
+reverted to its pre-2.2 state, and timetables will now always run at the same
+*clock time*, e.g. 9am in the above example.
```

Comment:
It seems we should state that this was a bug that we are fixing, not simply a change that we are reverting.  We have observed that it's not simply that the schedule doesn't run at the correct clock time, but that the dag actually gets stuck and fails to run.

Additionally I'm a bit confused by the 9am / 10am example.  Are you saying that the behavior before this fix would be that the dag would run at 10am until DST transitions back, 6 months later? Or only the first run?